documents/models.py

```diff
@@ -209,3 +209,22 @@ def buyer_company_name(self):
     @property
     def is_invoice(self):
         return "Invoice" in self.document_type
+
+
+class DocumentSendLog(BaseAbstractModel):
+    """
+    Model to track emails to which documents for an order were sent.
+    This way we want to avoid sending duplicate mails.
+    """
+
+    email = models.EmailField()
```

Comment:
I would use `ArrayField` (to store a list of emails) here to have a single entry per order. It would require `get()` instead of `filter()` but probably it won't change too much. I'm just thinking out loud. 🤔

Reply:
I do not think we would gain anything from array list. Order id would be distinct in the table? There would be anyway several entries per order anyway.